scripts/okta/lib/helper.ts

```diff
@@ -288,6 +291,23 @@ export const getRedirectUrl = (
 		return `/set-password/${setPasswordToken}?${params.toString()}`;
 	}
 
+	// if we have a page parameter, we need to go to that page on gateway
+	if (page) {
+		// only allow certain pages
+		switch (page) {
+			// show a specific page on gateway
+			case 'register':
+				return `/${page}?${params.toString()}`;
+			// instantly start google or apple authentication through gateway
+			case 'google':
+			case 'apple':
+				return `/signin/${page}?${params.toString()}`;
+			default:
+				// if the page parameter is not one of the allowed pages, then break
+				break;
```

Comment:
Perhaps default should be the default path and then get rid of L312:
```suggestion
				return `/signin?${params.toString()}`;;
```